azure/Kqlmagic/my_aad_helper.py

```diff
@@ -92,9 +93,14 @@ def __init__(self, kcsb, default_clientid, **options):
         client_id = kcsb.application_client_id or default_clientid
         self._resource = "{0.scheme}://{0.hostname}".format(urlparse(kcsb.data_source))
         token_cache = None
-        isSso = "FALSE" # os.getenv("{0}_ENABLE_SSO".format(Constants.MAGIC_CLASS_NAME.upper()))
-        if (isSso and isSso.upper() == "TRUE"):
-            token_cache = AdalTokenCache()
+
+        SSO_encrypt_keys = AdalTokenCache.get_params_SSO()
+        if SSO_encrypt_keys[0] and options.get("enablesso"):
```

Comment:
you should not try to read SSO params if sso is disabled, to avoid exceptions or meessages about the parameters.